### ISSUE TITLE
Accessibility: Insufficient Color Contrast

### ISSUE THREAD

**Issue by HappyDoggo** [explorableexplanations/explorableexplanations.github.io#6]
Issues with:
* pretty much all the tags colors
* #gallery #entry #description
* with the rainbow tag, only the blue is lacking in contrast

**Comment by HappyDoggo** [comment]
Pull Request here: https://github.com/explorableexplanations/explorableexplanations.github.io/pull/7
